js/featherlight.js

```diff
@@ -21,25 +21,27 @@ Backdrop.behaviors.featherlight = {
     $.featherlight.prototype.afterContent = function() {
       var caption = false;
       var img = this.$currentTarget.find('img');
-      var key = 'id-' + img.attr('data-file-id');
-      var captionAttr = settings.featherlight.files[key];
-
-      if (typeof captionAttr !== 'undefined') {
-        // Caption attribute taken from field formatter settings.
-        if (captionAttr) {
-          var caption = img.attr(captionAttr);
+      if (img[0].hasAttribute('data-file-id')) {
+        var key = 'id-' + img.attr('data-file-id');
+        var captionAttr = settings.featherlight.files[key];
```

Comment:
The only error I can produce is: `TypeError: settings.featherlight.files is undefined`.